test/test_autograd.py

```diff
@@ -5307,7 +5307,7 @@ def run_functional_checks(test_case, test_name, name, apply_fn, run_grad_checks,
                 'mean', 'inverse', 'addcmul',
                 'addcdiv', 'linalg.tensorinv', 'matrix_exp',
                 'narrow', 'swapaxes', 'swapdims', 'tensor_split',
-                'baddbmm', 'addbmm', 'addmv'] + complex_list_filter + separate_complex_tests
+                'baddbmm', 'addmv'] + complex_list_filter + separate_complex_tests
```

Comment:
Hello @anjali411, I'm adding `OpInfo`s for `add`, `mm` and `addmm` via #55201.
Please advise as to whether I should remove `add` from this list in that PR.
Thanks!

Reply:
Why would you remove add, @imaginary-person ? This is for "add B mm".

Reply:
@mruberry, I'm adding an `OpInfo` for `add` in #55201, so I was asking if I should remove `add` from this list in that PR.

Reply:
Haha, sorry, that makes much more sense, @imaginary-person! Good question. @anjali411?

Reply:
@imaginary-person yeah the op name in the list is only used as long as the method_tests for that op exist, so yes, please remove the op names from the `complex_list` as you port the tests.